robot/Cumulus/resources/NPSP.py

```diff
@@ -1223,6 +1223,31 @@ def select_value_from_dropdown(self,dropdown,value):
         self.selenium.get_webelement(locator).click()
         self.wait_for_locator('popup')
         self.npsp.click_link_with_text(value)
+
+    @capture_screenshot_on_error
+    def click_rd2_modal_button(self, name):
+        """Based on the button name (Cancel)  or (Save) on the modal footer, select and click on the respective button"""
+        btnlocator = npsp_lex_locators["button-with-text"].format(name)
+        self.selenium.scroll_element_into_view(btnlocator)
+        ele = self.selenium.get_webelement(btnlocator)
+        self.selenium.click_element(ele)
+
+    @capture_screenshot_on_error
+    def select_value_from_rd2_modal_dropdown(self,dropdown,value):
+        """Selects given value in the dropdown field on the rd2 modal"""
+        locator = npsp_lex_locators["erd"]["modal_dropdown_selector"].format(
+            dropdown
+        )
+        selection_value = npsp_lex_locators["erd"][
+            "modal_selection_value"
+        ].format(value)
```

Comment:
just a cosmetic thing, suggest keeping the locator in single line. Looks much cleaner and nicer

Reply:
Tried readjusting